otcextensions/sdk/modelartsv1/v1/_proxy.py

```diff
@@ -9,9 +9,861 @@
+        """Delete Training Job Configuration.
+
+        :param name: Name of a training job configuration.
+        :param bool ignore_missing: When set to ``False``
+            :class:`~openstack.exceptions.ResourceNotFound` will be raised when
+            the config does not exist.
+            When set to ``True``, no exception will be set when attempting to
+            delete a nonexistent Training Job Configuration.
+
+        returns: ``None``
+        """
+        return self._delete(
+            _training_job_config.TrainingJobConfig,
+            name,
+            ignore_missing=ignore_missing,
+        )
+
+    # Visualization Job Management
+
+    def visualization_jobs(self, **params):
+        """List all Visualization Jobs.
+
+        :param dict params: Optional query parameters to be sent to limit
+            the visualization jobs  being returned.
+
+        :returns: a generator of :class:`~otcextensions.sdk.modelartsv1.v1.
+            visualization_job.VisualizationJob` instances.
+        """
+        return self._list(_visualization_job.VisualizationJob, **params)
+
+    def create_visualization_job(self, **attrs):
+        """Create a Visualization Job from attributes.
+
+        :param dict attrs: Keyword arguments which will be used to create a
+            :class:`~otcextensions.sdk.modelartsv1.v1.
+            visualization_job.VisualizationJob`
+            comprised of the properties on the Visualization Job class.
+
+        :returns: The results of Visualization Job creation.
+
+        :rtype:
+            :class:`~otcextensions.sdk.modelartsv1.v1.
+            visualization_job.VisualizationJob`.
+        """
+        return self._create(_visualization_job.VisualizationJob, **attrs)
+
+    def get_visualization_job(self, visualization_job):
+        """Get details of a Visualization Job.
+
+        :param visualization_job: visualization job id or an instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.
+            visualization_job.VisualizationJob`.
+
+        :returns: instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.
+            visualization_job.VisualizationJob`.
+        """
+        return self._get(
+            _visualization_job.VisualizationJob, visualization_job
+        )
+
+    def update_visualization_job(self, visualization_job, description):
+        """Update visualization job description.
+
+        :param visualization_job: visualization job id or an instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.
+            visualization_job.VisualizationJob`.
+        :param description: Description of a visualization job to update.
+
+        :returns: instance of :class:`~otcextensions.sdk.modelartsv1.v1.
+            visualization_job.VisualizationJob`
+        """
+        return self._update(
+            _visualization_job.VisualizationJob,
+            visualization_job,
+            job_desc=description,
```

Comment:
desc is similar to descending, better use full word

Reply:
for put/update operation we need to pass correct key name "job_desc" is key here :)
in function arguments its used as description.